pandas/core/frame.py

```diff
@@ -2211,6 +2212,8 @@ def to_html(
         border : int
             A ``border=border`` attribute is included in the opening
             `<table>` tag. Default ``pd.options.display.html.border``.
+        encoding : str
```

Comment:
hi @ailurus1991 , per [contributing_docstring#section-3-parameters](https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#section-3-parameters), the recommended way to document default values is `param : type, default v`

id suggest changing it to

```
encoding : str, default "utf-8"
        Set character encoding
```

Reply:
many thanks and I'll check contributing_docstring more carefully!